src/main.ts

```diff
@@ -120,9 +125,14 @@ export class Purchases {
    * keep the returned instance around for use throughout your application.
    * @param apiKey - RevenueCat API Key. Can be obtained from the RevenueCat dashboard.
    * @param appUserId - Your unique id for identifying the user.
+   * @param httpConfig - Advanced http configuration to customise the SDK usage {@link HttpConfig}.
    * @throws {@link PurchasesError} if the API key or user id are invalid.
    */
-  static configure(apiKey: string, appUserId: string): Purchases {
+  static configure(
+    apiKey: string,
+    appUserId: string,
+    httpConfig?: HttpConfig,
```

Comment:
In the mobile SDKs we have the concept of `DangerousSettings` for settings that are not meant to be used by developers unless specifically directed to like: https://github.com/RevenueCat/purchases-android/blob/main/purchases/src/main/kotlin/com/revenuecat/purchases/DangerousSettings.kt.

I'm wondering if we should add these now, and add the `HttpConfig` parameter like one of the fields there. This is mostly as a way to make it even clearer that devs shouldn't use this inadvertently. Wdyt?

Reply:
Do we consider the `proxy` field we have in other SDKs dangerous?

I think that these settings are as dangerous as the `proxy`.